OpenRA.Game/Network/ReplayRecorderConnection.cs

```diff
@@ -50,8 +50,7 @@ void StartSavingReplay(byte[] initialContent)
 			var mod = Game.ModData.Manifest.Mod;
 			var dir = Platform.ResolvePath("^", "Replays", mod.Id, mod.Version);
 
-			if (!Directory.Exists(dir))
-				Directory.CreateDirectory(dir);
+			Directory.CreateDirectory(dir);
```

Comment:
Because the check for the existance of directory is not necessary. If it exists, Directory.CreateDirectory() does nothing.